mistral_client.py

```
@@ -158,3 +158,42 @@ def test_connection(self):
+            }
+
+            response = requests.post(
+                f"{self.base_url}/chat/completions",
+                headers=headers,
+                json=payload,
+                timeout=15
+            )
+
+            if response.status_code == 200:
+                return response.json()['choices'][0]['message']['content'].strip()
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)